class.jetpack.php

```diff
@@ -535,8 +535,10 @@ private function __construct() {
 			add_action( 'init', array( 'Jetpack_Keyring_Service_Helper', 'init' ), 9, 0 );
 		}
 
-		$this->tracking = new Tracking( 'jetpack' );
-		add_action( 'init', array( $this, 'track_jetpack_usage' ) );
+		if ( self::jetpack_tos_agreed() ) {
+			$tracking = new Automattic\Jetpack\Plugin\Tracking();
```

Comment:
Did we agree on some kind of conventions for cases like this? When do we import all namespaces at the top of a file with `use`, and when do we use the namespace like here?

Reply:
we didn't but I think we prefer to use `use` at the top.